### ISSUE TITLE
Tasking Manager Message Forwarded to Email

### ISSUE THREAD

**Issue by MikeTho16** [hotosm/osm-tasking-manager2#865]
When a validator (or anyone) sends someone a message through the tasking manager, the message should forwarded to the recipient's email account.  It is too easy for an occasional mapper to miss messages, or see them days after the message was sent, by which time the work which is be the subject of the message has been forgotten.

**Comment by ethan-nelson** [comment]
Unfortunately this is difficult as we don't have access to the email from their OSM profile (where we initially get their profile info) or anywhere else. It would have to rely on users supplying a current email, which would need to be done after the initial authentication through OSM.

**Comment by MikeTho16** [comment]
Interesting.  Here is a possible work around.  We know their OSM name, can we forward the message through the OSM messaging system? e.g. http://www.openstreetmap.org/message/new/<user name> I think these are forwarded to the user's email address.

**Comment by ethan-nelson** [comment]
Yes they are forwarded, but unfortunately there's no way to access that through API. I couldn't find the issue in a quick search, but I think at one point someone mentioned making a link on the validation screen taking the validator directly to the new message screen on OSM directed at the mapper they are reviewing. That could be an alternative solution. Or maybe we do want to add emails to the TM user database.

**Comment by ethan-nelson** [comment]
Also, thanks for all the feedback you've posted! Still sifting through them..

**Comment by logrady** [comment]
As identified in today's Training Working Group Meeting this is an important issue that keeps coming up. We need to find a way to resolve this. Perhaps a workaround would be to request adding their email address (optional) when logging in to the TM.

**Comment by MikeTho16** [comment]
Glad to hear that I am not the only one that has this concern.  I have
conducted a number of mapathons over the past year. Often someone will
participate in a event several months after the previous event they
participated in.  In the meantime, I, or some other validator has left them
feedback. During the second event they jump right into mapping not even
noticing that they have messages. Even if they would read the messages, so
much time has elapsed they remember none of the details of the mapping they
had previously done.

Mike

**Comment by aawiseman** [comment]
this is related to #906
